Makefile

```diff
@@ -13,10 +13,10 @@ INSTALL_STAMP := $(VENV)/.install.stamp
 .PHONY: clean check lint format tests
 
 install: $(INSTALL_STAMP) $(COMMIT_HOOK)
-$(INSTALL_STAMP): $(PYTHON) dev-requirements.txt requirements.txt checks/remotesettings/requirements.txt
-	$(VENV)/bin/pip install -Ur requirements.txt
-	$(VENV)/bin/pip install --no-deps -Ur checks/remotesettings/requirements.txt
-	$(VENV)/bin/pip install -Ur dev-requirements.txt
+$(INSTALL_STAMP): $(PYTHON) requirements/dev.txt requirements/default.txt checks/remotesettings/requirements.txt
+	$(VENV)/bin/pip install -Ur requirements/default.txt
+	$(VENV)/bin/pip install -Ur checks/remotesettings/requirements.txt
```

Comment:
We want the `--no-deps` here because of kinto-signer. Otherwise it will pull the whole pyramid ecosystem :) I should have added a comment.

Reply:
Ah, I wondered why Pyramid was being pulled in. I'll put `--no-deps` in and re-do the constraints file.

I wonder if it would be possible to add some variant of `kinto-signer` like `kinto-signer[no-web]` that doesn't rely on Pyramid?

Reply:
> I wonder if it would be possible to add some variant of kinto-signer like kinto-signer[no-web] that doesn't rely on Pyramid?

I know that the opposite is possible (extra deps) but not sure if `[no-deps]` is feasible. The best would be to extract the utils of kinto-signer into an external package...